### DISCUSSION TITLE
Dual home manager issue

### DISCUSSION THREAD

**Discussion by tsmullins** [Security-Onion-Solutions/securityonion#7571] [Unsupported Versions] [answered]
We have a manager that is behind a VPN with a 10.16.x.x address.  This box also has a RFC1918 172.26.x.x.  We recently added a IDH node that needs to connect to the 172.16.x.x address.  Which it does for most things.  However, filebeat and Elastic still try to connect to the 10.16.x.x address.

How can we force Filebeat and Elasticsearch to use the 172.16.x.x address?

Thanks
Shane

**Comment by dougburks** [comment, marked as answer]
Looks like this was handled by our Support team.
